launcher/spec/launch_policy.go

```diff
@@ -218,7 +231,7 @@ func verifyMonitoringConfig(policy MonitoringType, spec MonitoringType) error {
 	case None:
 		// If policy is 'None', spec must also be 'None'.
 		if spec != None {
-			return fmt.Errorf("spec configured for %v but policy is none", spec)
+			return fmt.Errorf("spec configured for %v but policy is none", toString(spec))
```

Comment:
nit: you can make MonitotringType as a receiver type and implement the method `func (mt MonitoringType) String() string` instead.  Then doing `return fmt.Errorf("spec configured for %s but policy is none", spec)` should be just fine.